### PR TITLE
Obstacle course third

### PR DESCRIPTION
**Pull request by AkithMadurapperuma**
My code tells the robot to turn 90 degrees to the left, go forward to ram into the wood block, then back up to the initial position to then align itself for the parking position. 
1 - Turn from start point 90 degrees to the left (from -10, 55, 90°)
2 - Forward to point (48, 55, 0°)
3 - Back up to start point (-10, 55, 0°)
image

### CODE REVIEW

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/akithPath.java
Comment (kennedyrobotics1) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/akithPath.java:1]: Seems to be correct. Great Job!

**Comment by OLeg25685** [comment]
Looks good, great job!

**Review by kennedyrobotics1 — APPROVED** [review]

**Review by CharlesJfkRbt — APPROVED** [review]

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/akithPath.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package org.firstinspires.ftc.teamcode.drive.opmode;
+
+import com.acmerobotics.roadrunner.geometry.Pose2d;
+import com.acmerobotics.roadrunner.geometry.Vector2d;
+import com.acmerobotics.roadrunner.trajectory.Trajectory;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+
```
Comment (CubeOfCheese) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/akithPath.java]: don't need empty line here

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/akithPath.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    public void runOpMode() {
+
+        SampleMecanumDrive drive = new SampleMecanumDrive(hardwareMap);
+
+        drive.setPoseEstimate(new Pose2d(-10, 55, Math.toRadians(90)));
+
+        TrajectorySequence forwardToWoodBack = drive.trajectorySequenceBuilder(new Pose2d(-10, 55, Math.toRadians(0)))
+                .lineTo(new Vector2d(48, 55))
+                .build();
+        Trajectory backToStart = drive.trajectoryBuilder(new Pose2d(48, 59, Math.toRadians(0)))
```
Comment (CubeOfCheese) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/akithPath.java]: This trajectory starts at (48, 59) but the previous trajectory ends at (48, 55) ??

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/akithPath.java
Comment (CubeOfCheese) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/akithPath.java:1]: Your file should not be named after you. It should be named something like ObstacleCoursePath.java

**Review by CubeOfCheese — APPROVED** [review]

**Pull request merged**